terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/welcome/index.jsp

```diff
@@ -60,6 +60,10 @@
 			<td><a id="EL" href="${pageContext.request.contextPath}/el">EL
 					Function Test</a></td>
 		</tr>
+        <tr>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...